receiver/httpcheckreceiver/scraper.go

```diff
@@ -85,6 +87,21 @@ func (h *httpcheckScraper) scrape(ctx context.Context) (pmetric.Metrics, error)
 					h.mb.RecordHttpcheckStatusDataPoint(now, int64(0), h.cfg.Targets[targetIndex].Endpoint, int64(statusCode), req.Method, class)
 				}
 			}
+
+			if err == nil && len(h.cfg.Targets[targetIndex].Body) > 0 {
+				if body, err := io.ReadAll(resp.Body); err != nil {
+					h.settings.Logger.Error("failed to read response", zap.Error(err))
```

Comment:
Would be good to mention which target this error applies to, wouldn't it?

Reply:
Thanks @astencel-sumo, that's a great suggestion.